### ISSUE TITLE
Can we use routahe as a library?

### ISSUE THREAD

**Issue by thykka** [anttikon/routahe#14]
Currently, it seems routahe is only designed for CLI usage. It would be nice to be able to import and use it as a javascript dependency. However, trying to import it as a module fails.

**Comment by anttikon** [comment]
You are right, routahe is currently designed for CLI usage only.

However, I'll do some refactoring later this week and enable importing. 🏭

**Comment by anttikon** [comment]
I just published `routahe@0.2.15` and you may use it like this:

```
import {getRoute} from 'routahe'
getRoute({from: 'Kamppi', to: 'Kannistontie 2, vantaa' }).then(routes => console.log(routes))
```

Just let me know if you need some extra features, help or something like that. ☝️ Cheers!
